openmdao/docs/advanced_guide/implicit_comps/defining_icomps.rst

```diff
@@ -0,0 +1,154 @@
+
+
+
+Building the Circuit Group and Solving it with NewtonSolver
+***********************************************************
+
+We can combine the :code:`Resistor`, :code:`Diode`, and :code:`Node` into the circuit pictured above using a :ref:`Group <feature_grouping_components>`.
+Adding components and connecting their variables is all the same as what you've seen before in the :ref:`Sellar - Two Discipline <sellar>` tutorial.
+What is new here is the addition use of the nonlinear :ref:`NewtonSolver <nlnewton>` and linear :ref:`DirectSolver <directsolver>` to converge the system.
+
+In previous tutorials we had used a gradient free :ref:`NonlinearBlockGaussSeidel <nlbgs>` solver, but that won't work here.
+Just above we discussed that the :code:`Node` class does, and in fact can not, define its own :code:`solve_nonlinear` method.
+Hence, there would be no calculations for the GaussSeidel solver to iterate on.
+Instead we use the Newton solver at :code:`Circuit` level, which uses jacobian information compute group level updates for all the variables simultaneously.
+The Newton solver's use of that jacobian information is why we need to declare a linear solver in this case.
+
+.. note::
+    OpenMDAO provides a library of :ref:`linear solvers <feature_linear_solvers>` that are useful in different advanced scenarios.
+    For many problems, especially problems built from components with mostly scalar variables the :ref:`DirectSolver <directsolver>`
+    will be both the most efficient and the easiest to use.
+    We recommend you stick with :ref:`DirectSolver <directsolver>` unless you have a good reason to switch.
+
+
+.. embed-test::
+    openmdao.test_suite.test_examples.test_circuit_analysis.TestCircuit.test_circuit_plain_newton
+    :no-split:
+
+
+Modifying Solver Settings in Your Run Script
+***********************************************************
+
+In the above run script, we set some initial guess values: :code:`prob['n1.V']=10` and :code:`prob['n2.V']=1`.
+If you try to play around with those initial guesses a bit, you will see that convergence is really sensitive to
+the initial guess you used for *n2.V*.
+Here is a second run-script that uses the same :code:`Circuit` group we defined previously, but modifies some solvers settings and initial guesses.
+If we set the initial guess for :code:`prob['n2.V']=1e-3`, then the model starts out with a massive residual.
+It also converges much more slowly so althought we gave it more than twice the number of iterations it doesn't even get close to a converged answer.
+
+
+.. embed-test::
+    openmdao.test_suite.test_examples.test_circuit_analysis.TestCircuit.test_circuit_plain_newton_many_iter
+    :no-split:
+
+
+.. note::
+
+   You actually can get this model to converge. But you have to set the options for :code:`maxiter`=400 and :code:`rtol`=1e-100.
+   You need to set set the `rtol` value to be so low to prevent premature termination.
```

Comment:
the word `set` occurs twice

Reply:
also it doesn't render correctly: ```maxiter`=400 and :code:`rtol`=1e-100. You need to set set the `rtol value to be so low to prevent premature termination.```